desk/app/contacts.hoon

```diff
@@ -398,7 +398,7 @@
         --
     ::
     ^+  cor
-    =/  bas  /(scot %p our.bowl)/contact-store/(scot %da now.bowl)
+    =/  bas  /(scot %p our.bowl)/contact-store/(scot %da now.bowl)/$
     ?.  .^(? gu+bas)  cor
     =/  ful  .^(rolodex:legacy gx+(weld bas /all/noun))
```

Comment:
The %gu scry needs to change, but the %gx does not. You should revert the change to `bas` and just ` .^(? gu+(weld bas /$))`. This one's a pain ...

Reply:
ah yeah you're right ok